tests/storage/test_online_resize.py

```diff
@@ -297,9 +293,8 @@ def test_disk_expand_then_clone_fail(
             func=lambda: dv.instance.status.conditions,
         ):
             if any(
-                "The clone doesn't meet the validation requirements:"
-                " target resources requests storage size is smaller than the source" in condition["message"]
-                for condition in sample
+                "resources requests storage size is smaller than the source" in condition.get("message", "")
```

Comment:
i assume there is a condition type that this message appears in, if so get the condition message using the type using`get_condition_message`